tools/report-converter/codechecker_report_converter/cli.py

```diff
@@ -137,15 +121,17 @@ def main():
     if 'verbose' in args and args.verbose:
         LOG.setLevel(logging.DEBUG)
 
+    clean = True if 'clean' in args else False
```

Comment:
if we remove the `default=argparse.SUPPRESS,` wouldn't be the default value set to False so in that case this logic is not needed?

Reply:
As a general note we might not need the argparse.SUPPRESS default values in this script?